src/itensor.jl

```diff
@@ -1963,8 +1963,9 @@ map(f, x::ITensor) = itensor(map(f, tensor(x)))
 w .+= a .* v
 ```
 """
-axpy!(a::Number, v::ITensor, w::ITensor) = (w .+= a .* v)
-
+function axpy!(a::Number, v::ITensor, w::ITensor)
+  return itensor(data(w) .+= a .* data(v), inds(w))
```

Comment:
This will be wrong for sparse tensors like `BlockSparse` and `Diag`. We'll need to fix it in `NDTensors` directly.

Broadcasting on ITensors eventually calls `permutedims` in NDTensors so seems like that would be the place to fix it.